test/extensions/quic_listeners/quiche/platform/epoll_address_test_utils_impl.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+// NOLINT(namespace-envoy)
+
+// This file is part of the QUICHE platform implementation, and is not to be
+// consumed or referenced directly by other Envoy code. It serves purely as a
+// porting layer for QUICHE.
+
+#include <sys/socket.h>
+
+#include "envoy/network/address.h"
+
+#include "test/test_common/environment.h"
+
+namespace epoll_server {
+
+namespace {
+
+int addressFamilyUnderTestHelper() {
+  std::vector<Envoy::Network::Address::IpVersion> versions =
+      Envoy::TestEnvironment::getIpVersionsForTest();
```

Comment:
How about using std::find for clarity?

if (std::find(versions.begin(), versions.end(), Envoy::Network::Address::IpVersion::v4) {
  return AF_INET;
}

if (std::find(versions.begin(), versions.end(), Envoy::Network::Address::IpVersion::v6) {
  return AF_INET6;
}

return -1;

Reply:
hmm, my code in comment doesn't actually work, it should be 

if (std::find(versions.begin(), versions.end(), Envoy::Network::Address::IpVersion::v4) != versions.end()) {
  return AF_INET;
}
// Same for v6